docs/pages/installation.mdx

```diff
@@ -31,7 +31,7 @@ running Teleport on UNIX variants other than Linux \[1].
 
 | Operating System | `teleport` Daemon | `tctl` Admin Tool | `tsh` and Teleport Connect User Clients [2] | Web UI (via the browser) | `tbot` Daemon |
 | - | - | - | - | - | - |
-| Linux v2.6.23+ (RHEL/CentOS 7+, Ubuntu 14.04+, and Debian 8+) \[3] | yes | yes | yes | yes | yes |
+| Linux v2.6.23+ (RHEL/CentOS 7+, Ubuntu 16.04+, and Debian 8+) \[3] | yes | yes | yes | yes | yes |
```

Comment:
Why are we updating it to 16.04?

Reply:
We support 14.04, but not through apt installation. Teleport can run on 14.04 through manual `.deb` install or tarball install.